packages/dev/core/src/Rendering/fluidRenderer/fluidRenderer.ts

```diff
@@ -215,6 +216,11 @@ export class FluidRenderer {
         this._onEngineResizeObserver = this._engine.onResizeObservable.add(() => {
             this._initialize();
         });
+
+        const engine = this._engine;
+        if (engine.isWebGPU) {
+            this._shaderLanguage = ShaderLanguage.WGSL;
+        }

```

Comment:
@deltakosh 
I think I've seen a number of PRs where we have the option of forcing the use of GLSL: is this something we should be doing everywhere (and therefore, in the fluid renderer too)?

Reply:
This is valid mainly where the shader code is tweakable like materials for plugins, particles and such to prevent back compat issues